secfixes_tracker/models.py

```diff
@@ -363,15 +379,3 @@ def to_json(self):
             'maximumVersionOp': self.maximum_version_op,
             'cpeUri': self.cpe_uri,
         }
-
-
-@app.cli.command('export', help='Export JSON files.')
```

Comment:
I think we need this.
instead of exporting from views.py, can you add back this and check published artifact.

Reply:
also gitignore .idea/.

and 
check if we need to add back `flask import nvd` in  cron.yml 
commit: https://github.com/aquasecurity/secfixes-tracker/commit/b0b003e8a6b5af4ea06d14ee2639e23774a01582#diff-69b9772a7fe51bd7c367926862d89ca7d8949cc9832b3645e610a1ec5bfa1ddf